examples/api/step2/index.ts

```diff
@@ -15,14 +15,18 @@
 import * as aws from "@pulumi/aws";
 
 const api = new aws.apigateway.x.API("myapi", {
-    routes: [
-        { method: "GET", path: "/b", handler: async (event) => {
-            return {
-                statusCode: 200,
-                body: "<h1>Hello world!</h1>",
-            };
-        }},
-    ],
+    routes: {
+        "/b": {
```

Comment:
Hmm - this isn't what I was imagining we would do here.  I thought we were going to keep `routes` as a single list and then just have entries for each kind of route.  Any reason we can't or don't want to do that?